frontend/components/PrintAreaButton/index.js

```diff
@@ -0,0 +1,41 @@
+import "./index.css";
+
+class PrintAreaButton {
+  constructor(node) {
+    this.node = node;
+    this.areaToPrint = node.getAttribute("data-area-to-print");
+    this.printableAreas = document.querySelectorAll("[data-print-area]");
+    this.printArea = this.printArea.bind(this);
+    this.addListener = this.addListener.bind(this);
+
+    this.addListener();
+  }
+
+  addListener() {
+    this.node.addEventListener("click", this.printArea);
+  }
+
+  printArea() {
```

Comment:
If I'm reading this correctly then if I just try and print the page without clicking the button I it will fail.

Reply:
Pressing `cmd+p` will still work however there is a bug where if you choose a particular area to print (ie the consent form) then press `cmd+p` it will only shown that chosen area.

This features in the current version of the consent form.

I have created #292 to fix this.